client/my-sites/hosting/site-backup-card/index.js

```diff
@@ -44,42 +54,80 @@ const SiteBackupCard = ( { disabled, lastGoodBackup, requestBackups, siteId, sit
 		? moment.utc( lastGoodBackup.last_updated, 'YYYY-MM-DD hh:mma' ).local().format( 'LLL' )
 		: null;
 
-	return (
-		<HostingCard className="site-backup-card">
-			<HostingCardHeading title={ translate( 'Site backup' ) }>
-				<HostingCardLinkButton
-					to={
-						wpcomAdminInterface === 'wp-admin'
-							? `https://cloud.jetpack.com/backup/${ siteSlug }`
-							: `/backup/${ siteSlug }`
-					}
-				>
-					{ translate( 'See all backups' ) }
-				</HostingCardLinkButton>
-			</HostingCardHeading>
-			{ hasRetrievedLastBackup && lastGoodBackup && ! isLoading && ! disabled && (
+	const renderContent = () => {
+		if ( ! hasBackup ) {
+			return (
 				<>
-					<p className="site-backup-card__date">
-						{ translate( 'Last backup was on:' ) }
-						<strong>{ lastGoodBackupTime }</strong>
+					<p>
+						<strong>{ translate( "Your plan doesn't support backups!" ) }</strong>
 					</p>
 					<p>
 						{ translate(
-							"If you restore your site using this backup, you'll lose any changes made after that date."
+							'Unlock more granular control over your site, with the ability to restore it to any previous state, and export it at any time'
```

Comment:
Missing punctuation.

```suggestion
							'Unlock more granular control over your site, with the ability to restore it to any previous state, and export it at any time.'
```